### PR TITLE
add fb app ID, improve social sharing descriptions

### PR DESCRIPTION
**Pull request by vietanhdev**
I added FB app ID option, edited some social sharing details so that it can be better.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -49,32 +49,69 @@
...
   <meta property="og:url" content="{{ site.url }}{{ page.url }}" />
+  <link rel="canonical" href="{{ site.url }}{{ page.url }}" />
   {% else %}
   <meta property="og:url" content="{{ site.url }}{{ page.url | remove: '/index.html' | remove: '.html' }}" />
+  <link rel="canonical" href="{{ site.url }}{{ page.url | remove: '/index.html' | remove: '.html' }}" />
   {% endif %}
+
   {% if page.share-img %}
   <meta property="og:image" content="{{ page.share-img }}" />
-  {% elsif site.avatar %}
```
Comment (daattali) [inline _includes/head.html]: Why did you remove this line? If there's no avatar, then we shouldn't serve {{site.url}} as the image

Comment (daattali) [inline _includes/head.html]: Great commit otherwise

Comment (vietanhdev) [inline _includes/head.html]: Sorry I've fixed it. Can you review it?

File: _config.yml [hunk 2]
```diff
@@ -19,6 +19,14 @@ title: My website
 # Short description of your site
 description: A virtual proof that name is awesome!
 
+# --- Misc --- #
+
+# Fill in your Google Analytics ID to track your website using GA
+# google_analytics: "UA-78252451-1"
```
Comment (daattali) [inline _config.yml]: could you remove this google_analytics line (since it already exists below)

File: _config.yml [hunk 3]
```diff
@@ -19,6 +19,14 @@ title: My website
...
+
+# Facebook App ID
+# fb_app_id: "1794997907410000"
```
Comment (daattali) [inline _config.yml]: could you move this line below to the Misc section, and remove your app ID please?

Comment (vietanhdev) [inline _config.yml]: Okey. I've done this.

**Comment by daattali** [comment]
thank you!
